### PR TITLE
Enhance visual script, selection area popped up when disconnected nodes

### LINKED ISSUE #11641: Visualscript disconnecting lines which node changes size cause some issues
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Godot 3.0 master https://github.com/godotengine/godot/commit/da144fed4c49133a93b3b83cfc665352c0af351a

**Issue description:**

**Steps to reproduce:**
1. Disconnect a line which node changes size
2. Enjoy issues

### PR DESCRIPTION
**Pull request by ChristopheLY**
*Bugsquad edit:* Fixes #11641.

### CODE REVIEW

File: scene/gui/graph_edit.cpp [hunk 1]
```diff
@@ -448,7 +448,7 @@ void GraphEdit::_top_layer_input(const Ref<InputEvent> &p_ev) {
 							}
 						}
 					}
-
+
```
Comment (akien-mga) [inline scene/gui/graph_edit.cpp:451]: Trailing whitespace.

File: modules/visual_script/visual_script_editor.cpp [hunk 2]
```diff
@@ -3101,7 +3102,7 @@ void VisualScriptEditor::_graph_connected(const String &p_from, int p_from_slot,
 	undo_redo->commit_action();
 }
 
-void VisualScriptEditor::_graph_disconnected(const String &p_from, int p_from_slot, const String &p_to, int p_to_slot) {
+void VisualScriptEditor::_graph_disconnected(const String &p_from, int p_from_slot, const String &p_to, int p_to_slot, bool is_pressed) {
```
Comment (akien-mga) [inline modules/visual_script/visual_script_editor.cpp:3105]: The parameter should be `p_is_pressed`.

**Comment by akien-mga** [comment]
CC @swarnimarun @fire @kidrigger

**Review by swarnimarun — CHANGES_REQUESTED** [review]
If there's a need for them feel free to explain but I don't think there is a need for calling the function multiple times.

**Comment by swarnimarun** [comment]
I still need to test the actual patch will do that later though.

**Comment by aaronfranke** [comment]
@ChristopheLY Is this still desired? If so, it needs to be [rebased](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) and tested on the latest master branch. Additionally, the commits need to be squashed into one (or few).

**Comment by DavidCambre** [comment]
I do not experience the source bug 11641

**Comment by akien-mga** [comment]
Closing as [we just removed VisualScript for 4.0](https://github.com/godotengine/godot/pull/64822), so this can no longer be evaluated as is.

The VisualScript module has been moved to https://github.com/godotengine/godot-visual-script with the aim that it might be ported to a GDExtension if there are interested contributors to do so. Further work to improve the features could be done on that repository too (so currently there isn't anyone dedicated to maintain this repository, it will depend again on whether some contributors step up to do so).

**Pull request closed without merging**